File: Day-11/Pattern12.js
function PatternTwelve(n) {
  let i = 1;
  while (i <= n) {
    let j = 1;
    let str = "";
    let ch = 65;
    while (j <= n) {
      str += String.fromCharCode(ch);
      j++;
      ch++;
    }
    i++;
    console.log(str);
  }
}

PatternTwelve(3);

File: Day-11/Pattern12.py
n = int(input())

def PatternTwelve(n):
    i = 1
    while(i <= n):
        ch = 65
        j = 1
        while(j <= n):
            print(chr(ch), end='')
            ch += 1
            j += 1
        print(end='\n')
        i += 1

PatternTwelve(n)